### DISCUSSION TITLE
Multiple windows for the same VisiData session

### DISCUSSION THREAD

**Discussion by joachimmaes** [saulpw/visidata#1403] [Ideas]
Is it possible to open multiple (OS-level) windows or terminals on the same VisiData session?

I have searched for this feature, but have not found what I was looking for.  Split view gives me the option to show sheets side by side, however I would like to leverage the capabilities of tmux or my window manager to get an overview more than 2 sheets simultaneously.

I am not sure if this feature makes sense to other users—or maybe there are other ways to approach this problem?

**Comment by geekscrapy** [comment]
I would absolutely bite for this, it's been in the back of my mind of how to do it for a while.

I have also been thinking about another example which is similar: have another window which is populated with the row, but vertical. Each time you move to another row, the other window is updated to show the current row. This would be useful as there are only a certain amount of columns that can be shown, and I think that another window with a vertical layout of only that current row, would be great.
